### PR TITLE
Run3-alca205E Update CaloOnlineTools/HcalOnlineDb for esConsume

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update CaloOnlineTools/HcalOnlineDb for esConsume

#### PR validation:

Useo the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by tvami** [comment]
@cmsbuild , please test

File: CaloOnlineTools/HcalOnlineDb/interface/HcalLutGenerator.h [hunk 1]
```diff
@@ -10,6 +10,8 @@
 #include "CalibFormats/HcalObjects/interface/HcalDbService.h"
 #include "CalibFormats/HcalObjects/interface/HcalDbRecord.h"
 #include "CondFormats/HcalObjects/interface/HcalChannelQuality.h"
+#include "CalibFormats/CaloTPG/interface/CaloTPGTranscoder.h"
+#include "CalibFormats/CaloTPG/interface/CaloTPGRecord.h"
 #include "CondFormats/DataRecord/interface/HcalChannelQualityRcd.h"
 
 class HcalLutGenerator : public edm::EDAnalyzer {
```
Comment (tvami) [inline CaloOnlineTools/HcalOnlineDb/interface/HcalLutGenerator.h:17]: As caught by the SA
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-125c12/19392/llvm-analysis/report-ea83a0.html#EndPath
this should be migrated away from being a legacy module

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by tvami** [comment]
+db

**Comment by qliphy** [comment]
+1

**Pull request merged**